### PR TITLE
Add AWS Marketplace to the pricing FAQ

### PR DESCRIPTION
**Pull request by iangrunt**

### CODE REVIEW

**Review by brikis98 — COMMENTED** [review]
This is great!

A couple minor suggestions:

Could you also mention using AWS Marketplace & AWS credits to the "fine print" on the `/pricing` page:

Screen Shot 2022-08-02 at 10 25 43 AM

And could you add that same "fine print" below the "Need help deciding?" box on the `/checkout` page?

Thanks!

File: _data/faq.yml [hunk 1]
```diff
@@ -224,13 +224,26 @@
 
 - category_name: Pricing
   items:
+    - question: What payment options do you accept?
+      answer: |
+        <p>
+          We accept AWS Marketplace for annual and multi-year contracts. For monthly billing, we require automated payments via credit card or ACH transfer. If you wish to use invoices and pay by wire transfer or check instead, you must do annual billing.
```
Comment (josh-padnick) [inline _data/faq.yml:230]: Might want to make it more clear that AWS Marketplace is _an_ option but not necessarily the first or primary option.

```suggestion
          For monthly billing, we require automated payments via credit card or ACH transfer. If you wish to use invoices and pay by wire transfer or check instead, you must do annual billing. We also accept AWS Marketplace for annual contracts.
```

File: pages/checkout/_need-help.html [hunk 2]
```diff
@@ -5,4 +5,14 @@ <h2 class="need-help-text">Need help deciding?</h2>
       >Contact Sales</a
     >
   </div>
+<div class="row row-page-header padding-top-none">
+  <div class="col-xs-12 col-md-8 col-md-offset-2">
+    <p class="text-center text-muted small">
+      Discounted pricing is available for non-profits, educational institutions,
+      and startups with fewer than 10 employees. Additionally, use your AWS
+      credits to purchase Gruntwork through our Marketplace listing.
+      <a href="/contact/">Contact us</a> to get the discounted pricing.
```
Comment (brikis98) [inline pages/checkout/_need-help.html:14]:
```suggestion
      Discounted pricing is available for non-profits, educational institutions,
      and startups with fewer than 10 employees. Additionally, you can use AWS
      credits to purchase a Gruntwork subscription through our AWS Marketplace listing.
      <a href="/contact/">Contact us</a> for the details.
```

File: pages/checkout/_need-help.html [hunk 3]
```diff
@@ -5,4 +5,14 @@ <h2 class="need-help-text">Need help deciding?</h2>
       >Contact Sales</a
     >
   </div>
+<div class="row row-page-header padding-top-none">
```
Comment (brikis98) [inline pages/checkout/_need-help.html:8]: Could you put this below and outside the "Need help deciding?" box? The same way it is below and outside the box on the `/pricing` page?

File: pages/pricing/_terms.html [hunk 4]
```diff
@@ -2,7 +2,8 @@
   <div class="col-xs-12 col-md-8 col-md-offset-2">
     <p class="text-center text-muted small">
       Discounted pricing is available for non-profits, educational institutions,
-      and startups with fewer than 10 employees.
+      and startups with fewer than 10 employees. Additionally, use your AWS
+      credits to purchase Gruntwork through our Marketplace listing.
       <a href="/contact/">Contact us</a> to get the discounted pricing.
```
Comment (brikis98) [inline pages/pricing/_terms.html:7]:
```suggestion
      and startups with fewer than 10 employees. Additionally, you can use your AWS
      credits to purchase a Gruntwork subscription through our AWS Marketplace listing.
      <a href="/contact/">Contact us</a> for the details.
```

**Review by brikis98 — COMMENTED** [review]
A few minor wording/styling tweaks and this is good to go!

File: pages/checkout/_need-help.html (hunk 3 above)
Comment (iangrunt) [reply pages/checkout/_need-help.html:9]: Whoops, great catch. I don't have a great local setup for this probably should get that going!

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by brikis98** [comment]
Thanks Ian! Don't forget to create a release to actually deploy the changes: https://github.com/gruntwork-io/gruntwork-io.github.io/releases

**Comment by iangrunt** [comment]
> Thanks Ian! Don't forget to create a release to actually deploy the changes: https://github.com/gruntwork-io/gruntwork-io.github.io/releases

Ah yes, thank you! Should be all good now.

**Comment by brikis98** [comment]
Excellent, great to see it live. Thanks for doing that @iangrunt!

**Pull request merged**